sim/ofat.py

```diff
@@ -121,6 +124,33 @@ def analyse(root_path):
         "tp": tps
     })
 
+def plot(root_path):
+
+    utils.assert_path_exists(root_path)
+
+    dirs = glob.glob(root_path + "/*/stats.csv")
+
+    values = pd.DataFrame(index =[range(1, len(dirs)+1)])
+    branches=['branch:']
+
+    for directory in dirs:
+        path = Path(directory)
+        branch_name = path.parts[-2]
+        branches.append(sample_name)
```

Comment:
Need to change to branch name in all other instances, e.g. on this line too

Reply:
Ah right missed that!